Translations/pl.po

```diff
@@ -7966,16 +7934,12 @@ msgstr ""
 
 #. TRANSLATORS: Neutral dialog spoken by Farnham (Gossip)
 #: Source/textdat.cpp:411
-#, fuzzy
-#| msgid ""
-#| "Griswold? Good old Griswold. I love him like a brother! We fought "
-#| "together, you know, back when... we... Lazarus... Lazarus... Lazarus!!!"
 msgid ""
 "Griswold? Good old Griswold. I love him like a brother! We fought together, "
 "you know, back when... we... Lazarus...  Lazarus... Lazarus!!!"
 msgstr ""
 "Griswold? Stary, dobry Griswold. Kocham go jak brata! Walczyliśmy razem, "
-"wiesz? Dawniej, kiedy... my... Lazarus... Lazarus... Lazarus!!!"
+"wiesz? Dawniej, kiedy... my... Lazarus...  Lazarus... Lazarus!!!"
```

Comment:
```suggestion
"wiesz? Dawniej, kiedy... my... Lazarus... Lazarus... Lazarus!!!"
```

Reply:
This is the hunk that I was not so sure about -- the source text has two spaces after the first "Lazarus..." and I wanted to match that, but I'm also OK with a single space being here in the translation.